BanchoBot/functions/tournaments/matchup/runMatchup.ts

```diff
@@ -617,6 +622,13 @@ async function runMatchupListeners (matchup: Matchup, mpLobby: BanchoLobby, mpCh
             }
         } else 
             mapsPlayed.push(beatmap);
+
+        // Panic if lobby is empty
+        if (playersInLobby.length === 0) {
+            await panic();
+            return;
```

Comment:
we might not want to return-early here. not sure what `playersPlaying = playersInLobby;` would change, but i think we most definitely want the centrifugo event so that all displays are in sync with the actual state of the lobby, no?

Reply:
considering https://github.com/Corsace/corsace/blob/d13fc9ffc73782ac3c08d73288fa2d14eac0fb86/BanchoBot/functions/tournaments/matchup/runMatchup.ts#L615 actually im thinking now of calling abortmatch before the panic here, and then adding the panic into that if statement as well

Reply:
Did the thing i said, tested it with empty lobby and it does abort and panic